backend/tracim_backend/models/event.py

```diff
@@ -19,6 +20,32 @@
 from tracim_backend.models.meta import DeclarativeBase
 
 
+class EventType(object):
+    """
+    Helper to convert event_type string as real used parameters in
+    database
+    """
+
+    def __init__(self, name: str) -> None:
+        event_type_data = name.split(".")
+        if not len(event_type_data) in [2, 3]:
+            raise ValidationError("event_type should have 2 to 3 parts")
+        if len(event_type_data) == 2:
+            event_type_data.append(None)
+        entity = event_type_data[0]
+        operation = event_type_data[1]
+        subtype = event_type_data[2]
+
+        if entity not in EntityType.values():
```

Comment:
Only trying to create an EntityType with the value and catch/reraise the exception that is raised if the string does not represent an entity type would be simpler (and more pythonic :smiley: ).

Reply:
done.